### PR TITLE
disable chronyd after ntpd set up

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1651875

### CODE REVIEW

**Comment by rishabh-d-dave** [comment]
Correct. I'll add a check. :)

**Comment by rishabh-d-dave** [comment]
@leseb @guits can you please review this PR?

File: roles/ceph-common/tasks/main.yml [hunk 1]
```diff
@@ -64,6 +64,31 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
+- name: check and disable chronyd
+  block:
+    - name: check if chronyd is installed on debian based systems
+      command: dpkg -l chronyd
+      register: chronyd_deb_installed
+      ignore_errors: true
```
Comment (leseb) [inline roles/ceph-common/tasks/main.yml]: use 
`failed_when: false` instead so we don't pollute the logs

**Comment by guits** [comment]
@rishabh-d-dave is there a corresponding patch in master for this?

**Comment by rishabh-d-dave** [comment]
@guits Yep, PR #3439. The scope of it is a bit more broader since it also merges `ntp_xxx.yml`s and number of NTP daemons our codebase support on master is more than stable-3.1.

**Comment by leseb** [comment]
@rishabh-d-dave let me recap:

* **always** use the github label to trigger a backport
* if there is a conflict, fix it in the branch pushed by mergify
* **never** push directly in the stable branch, unless master has diverged so much that files don't exist anymore.

I'm closing this one, so we can wait for #3439 to pass in master and work on the backport.
Thanks for your understanding.

**Comment by rishabh-d-dave** [comment]
@leseb 
Sebatien Han wrote:
> @rishabh-d-dave let me recap:
>
>    always use the github label to trigger a backport
>    if there is a conflict, fix it in the branch pushed by mergify
>    never push directly in the stable branch, unless master has diverged so much that files don't exist anymore.

Honestly, I wasn't aware of these except for that first one (that too only recently when Guillaume). Am I missing reading some contribution guideline? Because `CONTRIBUTING.md` had nothing backport.

>
> I'm closing this one, so we can wait for #3439 to pass in master and work on the backport.
Thanks for your understanding.

Sure. Thanks for informing me.  :)

**Pull request closed without merging**